appveyor.yml

```diff
@@ -55,6 +55,10 @@ environment:
     - JAVA_HOME: C:\Program Files\Java\jdk11
       DESC: "site, without verify (JDK11)"
       CMD: "./.ci/validation.cmd site_without_verify"
+    # site, git diff
+    - JAVA_HOME: C:\Program Files\Java\jdk11
+      DESC: "site, git diff"
+      CMD: "./.ci/validation.cmd git_diff"
```

Comment:
Move this back to "ps:"
Before `"Size of caches (bytes):"`

We need to see that appveyour is failing on first job

Reply:
Right.